not_tcp/not_tcp_test.py

```diff
@@ -57,13 +57,19 @@ async def driver(ctx):
         #
         # Wait for everything to be flushed:
         await ctx.tick().until(~dut.connected)
+
+        # TODO DO NOT MERGE
```

Comment:
Please fix before approval :)

Reply:
Done - works with the other/earlier fixes